LIMS2DB/classes.py

```diff
@@ -361,18 +362,13 @@ def build(self):
                     if 'Size (bp)' in agr_inp.udf_dict:
                         self.obj['projects'][project_luid]['samples'][sample_name]['library'][agr.luid]['size'] = round(agr_inp.udf_dict['Size (bp)'], 2)
 
-                    #Grabbing indexes
-                    # Get all artifacts for given sample
-                    query = "select art.* from artifact art \
-                        inner join artifact_sample_map asm on asm.artifactid=art.artifactid \
-                        inner join sample sa on sa.processid=asm.processid \
-                        where sa.processid = {sapid};".format(sapid=sample.processid)
+                    # Fetch index (reagent_label) information
                     try:
                         artifacts = self.session.query(Artifact).from_statement(text(query)).all()
```

Comment:
Wouldn't this then use the query on line 365/365 then?

Reply:
That query didn't return the correct indexes. I'm not sure why it was like that before.

Reply:
Ok, not too sure about what it should have done but I trust you 😄